project/src/components/place-card/place-card.tsx

```diff
@@ -5,14 +5,17 @@ import {AppRoute} from '../../const';
 type PlaceCardProps = {
   offers: OfferType;
   onActiveOfferChange: (offer: OfferType | null) => void;
+  onPlaceCardHover: (id: number) => void;
```

Comment:
```suggestion
  onActiveOfferChange: (offer: OfferType | null) => void;
```
тут нужно использовать один и тот же обработчик, тк он делает одно и то же